### PR TITLE
Display National Health ID before BP Passports in `EditPatientScreen`

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/6781/display-national-health-id-before-bp-passports-in-editpatientscreen

### CODE REVIEW

File: app/src/main/res/layout/screen_edit_patient.xml [hunk 1]
```diff
@@ -196,7 +197,6 @@
         android:text="@string/patientedit_add_button"
         android:textAlignment="viewStart"
         android:textAppearance="?attr/textAppearanceButton"
-        android:visibility="gone"
```
Comment (Honey14) [inline app/src/main/res/layout/screen_edit_patient.xml:199]: We need bp passport button to be gone by default, I have a story for this [here](https://app.shortcut.com/simpledotorg/story/6763/show-bp-passport-text-when-bp-passport-is-null-as-well) which will update this when feature flag is true for this feature. Can you undo this, please?

**Review by Honey14 — APPROVED** [review]
LGTM, left one comment, ptal

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM, Left one comment, PTAL

File: app/src/main/res/layout/screen_edit_patient.xml (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/res/layout/screen_edit_patient.xml:200]: oh good catch. I've already made this change tho in the feature flag PR. So we don't need to handle [this](https://app.shortcut.com/simpledotorg/story/6393/move-adding-health-ids-from-edit-patient-behind-a-feature-flag) anymore.

Comment (Honey14) [reply app/src/main/res/layout/screen_edit_patient.xml:200]: We'll still need to remove this from this PR, if you've added in that and that PR is merged in, here you're removing it again

**Review by Honey14 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/screen_edit_patient.xml (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/res/layout/screen_edit_patient.xml:200]: yeah, I've changed that and pushed

File: app/src/main/res/layout/screen_edit_patient.xml [hunk 2]
```diff
@@ -197,9 +197,9 @@
...
         app:icon="@drawable/ic_add_circle_outline_24px"
         app:iconGravity="start"
+        android:visibility="gone"
```
Comment (janhavisinghh) [inline app/src/main/res/layout/screen_edit_patient.xml:202]: @Honey14 ^

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**